### PR TITLE
WIP - initial readme

### PR DESCRIPTION
**Pull request by JesperGreen**
PT Story: https://www.pivotaltracker.com/story/show/132155601

Changes proposed in this pull request:
- Updated README.md with hopefully relevant information

Screenshots: 
screen shot 2016-10-12 at 3 07 28 pm
screen shot 2016-10-12 at 3 07 39 pm

Ready for review!

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,44 @@
...
+  $ npm install
+  ```
+  ```
+  $ bower install
+  ```
+
+# Built with
+
+* [Ionic](https://ionic.io/)
+* [Node.js](https://nodejs.org/en/)
```
Comment (tochman) [inline README.md]: We are not really using Node. But you could point o Cordova

**Comment by tochman** [comment]
Any action on this?

**Comment by diraulo** [comment]
@JesperGreen have you been able to update the PR as per @tochman comments?

**Comment by tochman** [comment]
Shall we close this or is somebody actually going to make the requested changes?

**Comment by AmberWilkie** [comment]
I have no idea why all these files are turning up in this PR... Going to close this and open a new one.

**Pull request closed without merging**